memif-attach/main.go

```diff
@@ -83,23 +109,22 @@ func createMemif(ctx context.Context, conn api.Connection, socketID uint32, mode
 		log.Entry(ctx).Fatalln("ERROR: memifCreate failed:", err)
 	}
 	log.Entry(ctx).Infof("Memif interface created")
-	log.Entry(ctx).Infof("swIfIndex"+
-		"Role"+
-		"SocketID",
+	log.Entry(ctx).Infof("swIfIndex %v\n"+
+		"Role %v\n"+
+		"SocketID %v\n",
 		rsp.SwIfIndex, memifCreate.Role, memifCreate.SocketID)
-	// ifindex.Store(ctx, isClient, rsp.SwIfIndex)
 	return nil
 }
 
 func dumpMemif(ctx context.Context, conn api.Connection) (memif.RPCService_MemifDumpClient, error) {
 	c := memif.NewServiceClient(conn)
 
-	MemifDumpClient, MemifDumpErr := c.MemifDump(ctx, &memif.MemifDump{})
-	if MemifDumpErr != nil {
-		log.Entry(ctx).Fatalln("ERROR: MemifDump failed:", MemifDumpErr)
+	memifDumpMsg, err := c.MemifDump(ctx, &memif.MemifDump{})
+	if err != nil {
+		log.Entry(ctx).Fatalln("ERROR: MemifDump failed:", err)
```

Comment:
INFO[0000] Socket file dump
INFO[0000] Socket ID from dump:0

Reply:
`INFO[0000] Configuration file: "/etc/vpp/helper/vpp.conf" not found, using defaults 
vpp[204]: clib_elf_parse_file: open `linux-vdso.so.1': No such file or directory
INFO[0000] vpp[204]: clib_elf_parse_file: open `linux-vdso.so.1': No such file or directory  cmd=vpp
vpp[204]: clib_sysfs_prealloc_hugepages:260: pre-allocating 19 additional 2048K hugepages on numa node 0
vpp[204]: buffer: vlib_physmem_shared_map_create: pmalloc_map_pages: failed to mmap 19 pages at 0x1000000000 fd 5 numa 0 flags 0x11: Cannot allocate memory


INFO[0000] vpp[204]: clib_sysfs_prealloc_hugepages:260: pre-allocating 19 additional 2048K hugepages on numa node 0  cmd=vpp
vpp[204]: buffer: falling back to non-hugepage backed buffer pool
INFO[0000] vpp[204]: buffer: vlib_physmem_shared_map_create: pmalloc_map_pages: failed to mmap 19 pages at 0x1000000000 fd 5 numa 0 flags 0x11: Cannot allocate memory  cmd=vpp
INFO[0000]                                               cmd=vpp
INFO[0000] vpp[204]: buffer: falling back to non-hugepage backed buffer pool  cmd=vpp
INFO[0000] /var/run/vpp/api.sock was created after 208.991435ms 
vpp[204]: vat-plug/load: vat_plugin_register: oddbuf plugin not loaded...

INFO[0000] vpp[204]: vat-plug/load: vat_plugin_register: oddbuf plugin not loaded...  cmd=vpp
INFO[0000] Socket file created                          
INFO[0000] SocketID: 2
SocketFilename: /var/run/vpp/memif1.sock
IsAdd:true 
INFO[0000] Memif interface created                      
INFO[0000] swIfIndex 1
Role MEMIF_ROLE_API_SLAVE
SocketID 2 
INFO[0000] Socket file dump                             
INFO[0000] Socket ID from dump:0                        
FATA[0000] ERROR: connect to VPP master failed: dial unixpacket /var/run/vpp/memif1.sock: connect: no such file or directory 
`
current logs